moochub-schema.json

```diff
@@ -1,555 +1,3105 @@
+              "format": "iri",
+              "enum": [
+                "https://w3id.org/kim/hcrt/course"
+              ]
+            },
+            "type": {
+              "type": "string",
+              "enum": [
+                "Concept"
+              ],
+              "description": "The type of the object according to AMB standard."
+            },
+            "inScheme": {
+              "type": "string",
+              "format": "iri",
+              "description": "A pointer to the used scheme. Currently, only https://w3id.org/kim/hcrt/scheme is allowed.",
+              "enum": [
+                "https://w3id.org/kim/hcrt/scheme"
+              ]
+            }
+          }
         },
-        "next": {
-          "type": "string",
-          "format": "iri",
-          "description": "The next page if available"
+        "description": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Description of the course as an HTML document",
+          "example": "<p>A message on the Internet is sent through several networks and via different stations on its way to the target system. The individual stations are responsible for ensuring that the message is properly forwarded and finally delivered to the correct recipient. Each of these stations, if the message is sent in plain text, can receive the message and read its content. This means that a potential attacker, if he controls one of these intermediate systems, can also read the content of the message and even modify it before retransmitting it. Such attacks can have extreme effects on communication.</p>\\n\\n<p>In this course we will look at how and whether your connection to online banking is secure or whether the content of an e-mail is trustworthy. For this purpose we will deal with the basics of cryptography, security objectives and different types of encryption. In addition, we will provide insights into different models and standards that are used in practice.</p>\\n"
         },
-        "prev": {
-          "type": "string",
-          "format": "iri",
-          "description": "The previous page if available"
-        }
-      }
-    },
-    "data": {
-      "type": "array",
-      "description": "The actual data of the response",
-      "uniqueItems": true,
-      "items": {
-        "title": "Course",
-        "type": "object",
-        "description": "Serialized course object",
-        "required": [
-          "id",
-          "type",
-          "attributes"
-        ],
-        "properties": {
-          "id": {
+        "inLanguage": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "uniqueItems": true,
+          "description": "List of languages in which the course is offered. Each language should be abbreviated in accordance with BCP 47: https://tools.ietf.org/html/bcp47.",
```

Comment:
Not sure about this. Usually, ISO codes are pretty common. Is BCP 47 similar?

Reply:
BCP 47 is a standard maintained by the IANA and therefore reliable -> it is also strongly connected to ISO 639 series of standards